samples/hello/pom.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"
+		 tools:ignore="MissingTranslation">
```

Comment:
Which coding platform are you using? Because the indent in Notepad++ looks aligned with xsi:schemaLocation.

From: Wei Jia [mailto:notifications@github.com]
Sent: Tuesday, December 8, 2015 11:04 AM
To: AzureAD/azure-activedirectory-library-for-android <azure-activedirectory-library-for-android@noreply.github.com>
Cc: Heidi Jinxu Jia <jinjia@microsoft.com>
Subject: Re: [azure-activedirectory-library-for-android] Debugged two lint error (#459)


In samples/hello/pom.xml<https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/459#discussion_r46997466>:

> @@ -1,6 +1,7 @@

>  <?xml version="1.0" encoding="UTF-8"?>

>  <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

> -         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd<http://maven.apache.org/POM/4.0.0%20http:/maven.apache.org/maven-v4_0_0.xsd>">

> +         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd<http://maven.apache.org/POM/4.0.0%20http:/maven.apache.org/maven-v4_0_0.xsd>"

> +             tools:ignore="MissingTranslation">

nitpick: the indent for tools:ignore is weird, can we make it align with xsi:schemaLocation?

—
Reply to this email directly or view it on GitHub<https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/459/files#r46997466>.